### PR TITLE
Chore/status tabs chips FRT-423

### PR DESCRIPTION
**Pull request by bdebon**
# What does this PR do?

[> Link to the JIRA ticket](https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-423)

- Add flex-shrink on all icones
- Add stopping, starting, unknown

> Screenshot of the feature

Put screenshot following the template `before` => `after` here

---

### Global

- [x] This PR does not introduce any breaking change
- [ ] This PR introduces breaking change(s) and has been labeled as such
- [x] I have found someone to review this PR and pinged him

### Store

- [ ] This PR introduces new store changes

### NX

- [x] I have run the dep-graph locally and made sure the tree was clean i.e no circular dependencies
- [x] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`

### Clean Code

- [x] I made sure the code is type safe (no any)
- [ ] I have included a feature flag on my feature, if applicable

### CODE REVIEW

**Comment by evoxmusic** [comment]
A preview environment was automatically created via Qovery.  
Click on the link below to follow its deployment and use it.  
:point_right: [[PR] staging - Chore/status tabs chips FRT-423 - 2022-06-22T17:04:52Z](https://console.qovery.com/platform/organization/141c07c8-0dd9-4623-983b-3fdd61867255/projects/977bd069-20f7-4b24-bd95-28b44174cf43/environments/176b203b-0a8a-4d77-ad76-70309a89c99c/applications)

File: tailwind-workspace-preset.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ module.exports = {
     extend: {
       spacing: {
         'right-help-sidebar': '22.5rem',
+        'icon-size': '0.875rem',
```
Comment (RemiBonnet) [inline tailwind-workspace-preset.js:8]: Instead to use this, why you didn't use text-sm?

**Review by RemiBonnet — COMMENTED** [review]
LGTM to me 👍, but I have only one comment about icon size, thanks @bdebon !

File: tailwind-workspace-preset.js (hunk 1 above)
Comment (bdebon) [reply tailwind-workspace-preset.js:8]: You can't use a text-size value with width and height.
You can't do `w-text-sm`

**Pull request merged**